Core/src/org/sleuthkit/autopsy/communications/ContactsViewer.java

```diff
@@ -37,6 +37,7 @@
 import org.openide.util.lookup.ServiceProvider;
 import org.sleuthkit.autopsy.corecomponents.TableFilterNode;
 import org.sleuthkit.autopsy.directorytree.DataResultFilterNode;
+import org.sleuthkit.datamodel.BlackboardAttribute;
```

Comment:
The bottom part of the contacts viewer does not reset when all contact nodes are un-selected via ctrl+click